ingestion-beam/src/test/java/com/mozilla/telemetry/integration/PubsubIntegrationTest.java

```diff
@@ -130,7 +131,7 @@ public void canSendPubsubOutput() throws Exception {
 
     SinkOptions.Parsed sinkOptions = pipeline.getOptions().as(SinkOptions.Parsed.class);
     sinkOptions.setOutput(pipeline.newProvider(topicName.toString()));
-    sinkOptions.setOutputPubsubCompression(pipeline.newProvider(Compression.UNCOMPRESSED));
+    sinkOptions.setOutputPubsubCompression(StaticValueProvider.of(Compression.UNCOMPRESSED));
```

Comment:
I thought we changed this to use `pipeline.newProvider` because `StaticValueProvider.of` was causing issues? i mean, CI passes, so i'm fine with either, just confused.

Reply:
StaticValueProvider didn't cause problems that I'm aware of, but `pipeline.newProvider` should be more realistic since it will throw an error if you attempt to use a value before the pipeline is running.

There's something about this test that was failing with newProvider with Beam 2.13. I don't deeply understand the issue, but I'm assuming it's a bug in the DirectRunner or in the particular way we're manipulating options in the context of this test. I will add a comment to that effect.